docs/doc_tools/all_link_anchors_to_lowercase.py

```diff
@@ -20,7 +20,7 @@
 def list_all_html_files():
     html_files = list()
     for root, dirs, files in os.walk("../"):
-        # html_files.extend(filter(lambda x: x.endswith(".html"), files))
+        # html_files.extend(filter(lambda x: x. endswith(".html"), files))
```

Comment:
I know Python, and x.endswith should not contain a space, that would not follow convention.